combat-trainer.lic

```diff
@@ -1450,6 +1457,21 @@ class SpellProcess
     game_state.casting_weapon_buff = $weapon_buffs.include?(name)
     if data['ritual']
       cast_ritual(data, game_state)
+    elsif data['runestone_exists']
+      #DRC.message("runestone_exists setting is set")
```

Comment:
The debug statements are only there because if I need to adjust something over the testing I only want this debug information and not the whole script in debug. When ready for merging, I will remove these.